protocol-units/bridge/integration-tests/tests/client_movement_eth.rs

```diff
@@ -89,7 +89,7 @@ async fn test_movement_client_initiate_transfer() -> Result<(), anyhow::Error> {
 }
 
 #[tokio::test]
-#[ignore] // Test fail when run with the other test: https://github.com/movementlabsxyz/movement/issues/656
+//#[ignore] // Test fail when run with the other test: https://github.com/movementlabsxyz/movement/issues/656
```

Comment:
This passes locally now with `rust_backtrace=1 cargo test --test client_movement_eth -- --nocapture --test-threads=1` 👍